### PR TITLE
Reflect mirage-xen api change

### PR DESCRIPTION
**Pull request by TheLortex**
The PR that breaks API is the following: mirage/mirage-xen#16

### CODE REVIEW

File: lib_test/mirage/unikernel.ml [hunk 1]
```diff
@@ -29,27 +29,27 @@ module Server (C: Mirage_console_lwt.S) = struct
   let rec read_all c t =
     VX.read t
     >>= function
-    | Ok `Eof -> OS.Time.sleep_ns (Duration.of_sec 5) >>= fun () -> C.log c "EOF"
-    | Error _ -> OS.Time.sleep_ns (Duration.of_sec 5) >>= fun () -> C.log c "ERR"
+    | Ok `Eof -> Os_xen.Time.sleep_ns (Duration.of_sec 5) >>= fun () -> C.log c "EOF"
```
Comment (talex5) [inline lib_test/mirage/unikernel.ml]: Why is all this code changing to become Xen-specific? Shouldn't this code also run on e.g. Unix?

Comment (TheLortex) [reply lib_test/mirage/unikernel.ml]: Looking at `build.sh` it looks like the whole unikernel is dedicated to run on xen so I didn't bother using OS-independent modules.

Comment (talex5) [reply lib_test/mirage/unikernel.ml]: I don't have a strong opinion on this (it's just test / example code), but if it's easy to keep it portable that would be nice. If not, don't worry about it...

Comment (avsm) [reply lib_test/mirage/unikernel.ml]: I agree with @talex5 -- we might as well use `OS.Time` and save `Os_xen` for the actual Xen-specific modules.

**Comment by TheLortex** [comment]
I replaced back `Os_xen.Time` by `OS.Time` !

**Comment by talex5** [comment]
Travis says:
```
# File "xen/events_xen.ml", line 33, characters 13-33:
# Error: The module OS is an alias for module Os_xen, which is missing
```

**Comment by TheLortex** [comment]
Changes in `mirage-xen` haven't been upstreamed yet so this error is expected. You can take a look at https://github.com/mirage/mirage-xen/pull/16 first.

**Comment by avsm** [comment]
I've pushed a pin-depends to this branch that should let the CI turn green

**Pull request merged**